utils/after_success.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script is meant to be used by Travis-CI to publish artifacts (binary, source and javadoc jars) when releasing snapshots.
+# Right now it will be manually triggered to push jars to nexus server.
+
+SITE_VERSION=$(./gradlew properties | grep -o 'version.*' | sed -n -e 's/^.*version: //p')
```

Comment:
Garrett - the "-n" option to sed tells it to not print lines by default.  Then the "s/...//p" tells sed to print the result of the substitution if it does something.  The net result is that the only lines that sed outputs should be the ones starting with "version:" with the leading "version:\s*" stripped off.